### PR TITLE
Remove `cl--plist-remove' usage in telega stable branch

### PR DESCRIPTION
**Pull request by bit9tream**
Basically a backport of [this](https://github.com/zevlg/telega.el/commit/5e20c82918219cfc8555d17ebc6818c1d79fa62c) commit into `releases` branch.

### CODE REVIEW

File: telega-util.el [hunk 1]
```diff
@@ -930,7 +938,7 @@ SORT-CRITERIA is a chat sort criteria to apply. (NOT YET)"
                          (telega-sort-chats
                           (or sort-criteria telega-chat-completing-sort-criteria)
                           (telega-filter-chats (or chats telega--ordered-chats)
-                                               '(or main archive))))))
+                                               '(or main archive has-chatbuf))))))
```
Comment (zevlg) [inline telega-util.el:941]: Looks like unrelated

Comment (bit9tream) [reply telega-util.el]: Okay, removed this

**Comment by zevlg** [comment]
Merged in, thanks!
